fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/FHIRServerTestBase.java

```diff
@@ -890,4 +905,45 @@ private String getLink(Bundle bundle, String relation) {
         }
         return null;
     }
+
+    
+    /**
+     * Get the Kafka host/service name.
+     * @return String - The Kafka host/service name.
+     */
+    public String getAuditKafkaConnectionInfo() {
+        return auditKafkaConnectionInfo;
+    }
+
+    /**
+     * Get the Kafka topic name.
+     * @return String - the Kafka topic name.
+     */
+    public String getAuditKafkaTopicName() {
+        return auditKafkaTopicName;
+    }
+
+    /**
+     * Determines whether or not the server supports the audit logs
+     * examining the conformance statement.
+     */
+    protected boolean isAuditLogSupported() throws Exception {
+        Boolean auditLogSupported = Boolean.FALSE;
+        CapabilityStatement conf = retrieveConformanceStatement();
+        List<Extension> extensions = conf.getExtension();
+        String auditLogServiceName = null;
+        if (extensions != null) {
+            for (Extension extension : extensions) {
+                if (extension.getUrl().contains("auditLogServiceName")) {
+                    auditLogServiceName = extension.getValue().as(org.linuxforhealth.fhir.model.type.String.class).getValue();
```

Comment:
fyi we have a constant that we sometime statically import for this class:  `ModelSupport.FHIR_STRING`

Reply:
This has been fixed as suggested.